backend/db/migrations/20210717101512-addPolicies.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async () => {
+      await queryInterface.createTable('policies', {
+        id: {
+          type: Sequelize.INTEGER,
+          autoIncrement: true,
+          primaryKey: true
+        },
+        shop_id: {
+          type: Sequelize.INTEGER
+        },
+        all_policies: {
+          type: Sequelize.JSON
```

Comment:
nit: I think we favor using JSONB over JSON for postgres. though it is not support by sqlite. So in migrations we have been using the following pattern:
```type: isPostgres ? Sequelize.JSONB : Sequelize.JSON```
Let's use that as well here to be consistent?

Reply:
Sources online recommend using JSON over JSONB when the (associated) information in the database is only subject to 'retrieval and storage' [[1](https://sequelize.org/master/manual/other-data-types.html#note-for-postgresql), [2](https://stackoverflow.com/questions/22654170/explanation-of-jsonb-introduced-by-postgresql)]. Because that is true of this case, could we please forego code consistency for the sake of efficiency in storage?

Reply:
that's fine.